Mesh_3/include/CGAL/Mesh_3/C3T3_helpers.h

```diff
@@ -3470,11 +3474,19 @@ get_least_square_surface_plane(const Vertex_handle& v,
   // Compute least square fitting plane
   Plane_3 plane;
   Bare_point point;
-  CGAL::linear_least_squares_fitting_3(surface_point_vector.begin(),
-                                       surface_point_vector.end(),
+//  CGAL::linear_least_squares_fitting_3(surface_point_vector.begin(),
+//                                       surface_point_vector.end(),
```

Comment:
Is the `std::vector<Bare_point> surface_point_vector` still used?

Reply:
It's not but this draft is to be modified to satisfy periodic Mesh_3 needs, so I left as much info as possible for now